Sources/View/NibOwnerLoadable.swift

```diff
@@ -59,3 +59,15 @@ private extension NSLayoutConstraint {
   typealias Attribute = NSLayoutAttribute
 }
 #endif
+
+public extension NibOwnerLoadable where Self: UIViewController {
+    /**
+     Returns a `UIViewController` object instantiated from nib
+     
+     - returns: A `NibOwnerLoadable`, `UIViewController` instance
+     */
+    static func instantiate() -> Self {
+        let viewController = Self.init(nibName: String(describing: self), bundle: Bundle(for: self))
```

Comment:
Given [this behavior is actually already handled by iOS now](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621487-nibname) as calling `init` on UIViewController already makes iOS search for a XIB with the same name without you doing anything else, I don't think it adds much to have Reusable in play here. See also my comment in #45: I think we can close this PR since this behavior is provided by iOS already.